fast_api_als/services/verify_phone_and_email.py

```diff
@@ -14,16 +14,25 @@
 """
 
 async def call_validation_service(url: str, topic: str, value: str, data: dict) -> None:  # 2
+    start = int(time.time() * 1000.0)
+
     if value == '':
         return
     async with httpx.AsyncClient() as client:  # 3
         response = await client.get(url)
 
     r = response.json()
     data[topic] = r
+
+    time_taken = int(time.time() * 1000.0) - start
+
+    logging.basicConfig(filename='fast_api_als.log', filemode='w', format='%(name)s - %(levelname)s - %(message)s')
+    logging.info(f'Call Validation Time : {time_taken}')
```

Comment:
call validation for what?
be precise.

Reply:
Edited it to 'Phone and Email Validation Time'